lib/src/default_extension_map.dart

```diff
@@ -894,6 +894,7 @@ final Map<String, String> defaultExtensionMap = <String, String>{
   'wdp': 'image/vnd.ms-photo',
   'weba': 'audio/webm',
   'webm': 'video/webm',
+  'webmanifest': 'application/json',
```

Comment:
I would have expected

```suggestion
  'webmanifest': 'application/manifest+json',
```

https://w3c.github.io/manifest/#media-type-registration

@kevmoo - WDYT about me merging this PR with that change?

Reply:
LGTM! – points for putting that comment line IN the code so we track where all of these magic strings come from!